### ISSUE TITLE
More than 2 hours

### ISSUE THREAD

**Issue by ArtemSarmini** [jshackles/idle_master#204]
I want to idle some games in one time, but I already played a lot. Can I change this setting (2 hours limit) somewhere?

**Comment by claell** [comment]
It auto detects your playtime :)

**Comment by ArtemSarmini** [comment]
Look: I have more than 30 games with cards, but in each I played more than 2 hours and I can't use setting 'Idle games simultaneously up to 2 hours'. Can it be changed?

**Comment by claell** [comment]
You can use it. This setting however is only for getting playtime fast, since Steam used to only drop cards after two hours in the past. This has changed now afaik. If you are eligable to get cards you need to idle each game at once, instead drop rate gets decreased drastically.

**Comment by ArtemSarmini** [comment]
And I can't idle 20 games with 10h played in one IdleMaster window and must open 20 windows?

**Comment by claell** [comment]
No, you need to wait for each game to finish. After that the next game will be idled. Else the drop rate will get decreased.

**Comment by ArtemSarmini** [comment]
I understood, thanks.
